docs/tutorials/multimodal/index.rst

```diff
@@ -56,6 +56,12 @@ In the following, we prepared a few tutorials to help you learn how to use `Auto
 
       How to use CLIP to extract embeddings for retrieval problem.
 
+   .. card::
+      :title: Parameter-Efficient Finetuning in AutoMM -- Basic Usage
+      :link: efficient_finetuning_basic.html
+
+      How to take advantage of larger foundation models with the help of parameter-efficient finetuning.
```

Comment:
Is it better to highlight here (or in the title) that we use IA3 + BitFit? Because later we might have other tutorials that use more advanced parameter-efficient finetuning technique. But of course we could modify it at that time.

Reply:
Fixed and emphasized we are using IA^3 + BitFit.